tests/unit/disco/components/TestAddon.js

```diff
@@ -289,20 +284,24 @@ describe(__filename, () => {
     });
 
     it('does render a logo for an extension', () => {
-      const root = renderAddon({ addon: result });
+      const { heading, ...addon } = result;
+      const root = renderAddon({ addon, heading });
 
       expect(root.find('.logo')).toHaveLength(1);
     });
 
     it("doesn't render a theme image for an extension", () => {
-      const root = renderAddon({ addon: result });
+      const { heading, ...addon } = result;
+      const root = renderAddon({ addon, heading });
 
       expect(root.find('.Addon-ThemeImage-link')).toHaveLength(0);
       expect(root.find(ThemeImage)).toHaveLength(0);
     });
 
     it('throws on invalid add-on type', () => {
-      const root = renderAddon({ addon: result });
+      const { heading, ...addon } = result;
+      const root = renderAddon({ addon, heading });
```

Comment:
same here

Reply:
here, I need it because of the first expectation L305.

Reply:
yeh the test passes without the heading being passed in (I think)..I believe b/c it's in the default props..

yep, I did see L305 but it wasn't clear to me why that was there either 😕